content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -86,7 +86,8 @@ different Kubernetes components.
 | `DynamicAuditing` | `false` | Alpha | 1.13 | |
 | `DynamicKubeletConfig` | `false` | Alpha | 1.4 | 1.10 |
 | `DynamicKubeletConfig` | `true` | Beta | 1.11 | |
-| `EndpointSlice` | `false` | Alpha | 1.16 | |
+| `EndpointSlice` | `false` | Alpha | 1.16 | 1.16 |
+| `EndpointSlice` | `false` | Beta | 1.17 | |
```

Comment:
@robscott , should line 90 set the beta feature gate to `true`?

Reply:
Sadly no, we ended up switching it to disabled by default (https://github.com/kubernetes/kubernetes/pull/85365)

Reply:
OK. I was under the impression that all beta features were enabled by default.